src/app/courses/credits-course/course-credits.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+
+@Component({
+  templateUrl: './course-credits.component.html'
+})
+export class CourseCreditsComponent implements OnInit, OnDestroy {
+  id: string;
+  private sub: any;
+
+  constructor(private route: ActivatedRoute) {}
+
+  ngOnInit() {
+    this.sub = this.route.params.subscribe(params => {
+       this.id = params['id']; // (+) converts string 'id' to a number
```

Comment:
What do you mean by the comment at the end of the line?

Reply:
@paulbert Removed unnecessary comments. See https://github.com/ole-vi/planet/pull/248/commits/090143b6f9c919cb0ed6fc93bf7b5526a8e5ee67